app/components/commitfunds/TxConfirmation.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import { Alert } from "react-bootstrap";
+import { connect } from "react-redux";
+import { Link } from "react-router";
+
+import { IAppState } from "../../reducers/index";
+import { LoadingIndicator } from "../LoadingIndicator";
+import { UserInfo } from "../UserInfo";
+import { CommitHeaderComponent } from "./CommitHeaderComponent";
+
+import * as styles from "./TxConfirmation.scss";
+
+interface ITxConfirmationComponentProps {
+  errorMsg: string;
+}
+
+export class TxConfirmationComponent extends React.Component<ITxConfirmationComponentProps> {
+  public render() {
+    const errorMsg = this.props.errorMsg;
+
+    return (
+      <div>
+        <CommitHeaderComponent number="02" title="Confirm transaction" />
+        <UserInfo />
+        <p className={styles.text}>
+          Please confirm your transcation using signer or{" "}
+          <Link to="/commit">go back to change your commit</Link>.
```

Comment:
I'm not sure about this flow?
For injected web3 you will see overlay from parity / metamask and for ledger if you go back from this point transaction confirmation stays. If you ask for address probably you will block device.